package.json

```diff
@@ -65,10 +65,12 @@
     "test": "npm run cover"
   },
   "dependencies": {
+    "@google-cloud/automl": "^0.1.2",
     "@google-cloud/promisify": "^0.3.0",
     "google-gax": "^0.20.0",
     "is": "^3.2.1",
     "lodash.merge": "^4.6.1",
+    "mathjs": "^5.1.2",
```

Comment:
Guessing this was supposed to be in the samples package.json

Reply:
Fixed in latest commit.